### PR TITLE
Tax basis description storepickup

### LINKED ISSUE #5200: Question Tax basis in store pickup
Should Store be an option in the MODULE_SHIPPING_STOREPICKUP_TAX_BASIS code below
`    $db->Execute("insert into " . TABLE_CONFIGURATION . " (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, set_function, date_added) values ('Tax Basis', 'MODULE_SHIPPING_STOREPICKUP_TAX_BASIS', 'Shipping', 'On what basis is Shipping Tax calculated. Options are<br>Shipping - Based on customers Shipping Address<br>Billing Based on customers Billing address<br>Store - Based on Store address if Billing/Shipping Zone equals Store zone', '6', '0', 'zen_cfg_select_option(array(\'Shipping\', \'Billing\'), ', now())");`
It is in the description but omitted from the options.
either remove 
`<br>Store - Based on Store address if Billing/Shipping Zone equals Store zone`
or change to
`'zen_cfg_select_option(array(\'Shipping\', \'Billing\',\'Store\')`

Please do not change until #5199 is merged.

### PR DESCRIPTION
**Pull request by brittainmark**
fix #5200

### CODE REVIEW

File: zc_install/sql/updates/mysql_upgrade_zencart_158.sql [hunk 1]
```diff
@@ -224,6 +224,10 @@ UPDATE configuration SET val_function = '{"error":"TEXT_EMAIL_ADDRESS_VALIDATE",
 ### Added v158 bring banners_titles up to date
 UPDATE banners SET banners_title = 'Zen Cart Certified Services' WHERE banners_html_text LIKE '%<script>%';
 
+#############
+### Added to correct storpicku tax basis
```
Comment (scottcwilson) [inline zc_install/sql/updates/mysql_upgrade_zencart_158.sql]: please correct typo

**Comment by scottcwilson** [comment]
No change needed to includes/classes/order.php line 691 ?

**Comment by brittainmark** [comment]
Ok see what you mean. Will need the description changing to reflect this. Will redo with correct descriptions.

**Comment by brittainmark** [comment]
Corrected to amend the description.

**Review by scottcwilson — CHANGES_REQUESTED** [review]

File: zc_install/sql/updates/mysql_upgrade_zencart_158.sql (hunk 1 above)
Comment (brittainmark) [reply zc_install/sql/updates/mysql_upgrade_zencart_158.sql]: Have corrected. storpicku to storepickup.

**Pull request merged**